packages/beacon-node/test/e2e/eth1/jsonRpcHttpClient.test.ts

```diff
@@ -148,7 +153,16 @@ describe("eth1 / jsonRpcHttpClient", function () {
       const controller = new AbortController();
       if (abort) setTimeout(() => controller.abort(), 50);
       const eth1JsonRpcClient = new JsonRpcHttpClient([url], {signal: controller.signal});
-      await expect(eth1JsonRpcClient.fetch(payload, {timeout})).to.be.rejectedWith(error);
+
+      try {
+        await eth1JsonRpcClient.fetch(payload, {timeout});
```

Comment:
@nazarhussain try/catch is problematic here, test case would pass if no error is thrown

Either have to do explicit fail if there is no error like it is done here
https://github.com/ChainSafe/lodestar/blob/cdca9fbd589e450bcdb190032e97e16ea4ba0f13/packages/cli/test/unit/validator/decryptKeystoreDefinitions.test.ts#L77

or can use `.to.be.rejected.then` pattern

Reply:
Yes I knew that pattern, but we discussed earlier to not use the `nested then/catch` and prefer `try/catch`.

Reply:
In that case need to explicitly fail the test if there is no error